### PR TITLE
feat(cert-manager-webhook): support external defined secret

### LINKED ISSUE #21: [scaleway-certmanager-webhook] can't use secret defined outside of chart
Hello,

Currently you have to pass access key and secret key to the helm chart even if you just want to reference an external secret defined in an other way using for example external secret operator with vault.

I will open a PR, regarding this issue.

### PR DESCRIPTION
**Pull request by M0NsTeRRR**
Fixes #21

### CODE REVIEW

File: charts/scaleway-certmanager-webhook/templates/deployment.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ spec:
             {{- with .Values.extraEnv }}
             {{- toYaml . | nindent 12 }}
             {{- end }}
-          {{- if and .Values.secret.accessKey .Values.secret.secretKey }}
+          {{- if .Values.secret.name }}
```
Comment (Nox-404) [inline charts/scaleway-certmanager-webhook/templates/deployment.yaml:43]: doing so you need to remove the default name in order not to mount a secret, which breaks the default behavior

Comment (Nox-404) [reply charts/scaleway-certmanager-webhook/templates/deployment.yaml:43]: maybe add another flag or something to set external secret ?

Comment (M0NsTeRRR) [reply charts/scaleway-certmanager-webhook/templates/deployment.yaml:43]: Yes you are right, adding a new option could be sufficient for this case, I will edit the PR next week :)

Comment (M0NsTeRRR) [reply charts/scaleway-certmanager-webhook/templates/deployment.yaml]: @Nox-404 it should be better now :)

**Review by Nox-404 — APPROVED** [review]
lgtm

**Pull request merged**